front/src/modules/trainschedule/components/ManageTrainSchedule/Map.tsx

```diff
@@ -70,6 +70,8 @@ function Map() {
     bottom: 20,
   };
 
+  const { updateFeatureInfoClick } = useOsrdActions();
```

Comment:
Could you group it with the other custom hook line 51 :)

Reply:
Done